### PR TITLE
Fix/merch products

### PR DESCRIPTION
**Pull request by Iyzyman**
- Fixing products page as per requested in merch-products/ivan [pr](https://github.com/ntuscse/website/pull/135)

### CODE REVIEW

File: apps/cms/src/@types/Product.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+//     public id: string = "",
+//     public name: string = "",
+//     public price: number = 0, // Price of the product in cents ($19 = 1900).
+//     public images: string[] = [], // URL of the images.
+//     public sizes: ProductSizeTypes[] = [],
+//     public productCategory: ProductCategoryType = "",
+//     public isAvailable?: boolean
+//   ) {}
+// }
+
```
Comment (limivann) [inline apps/cms/src/@types/Product.ts:12]: Can remove this lines for now

**Review by limivann — APPROVED** [review]
Overall, LGTM!

File: apps/cms/src/@types/Product.ts (hunk 1 above)
Comment (Iyzyman) [reply apps/cms/src/@types/Product.ts]: Done

**Review by chanbakjsd — APPROVED** [review]
Thanks for fixing the issues. LGTM!

**Pull request merged**